packages/aggregator/src/analytics/aggregateAnalytics/aggregations/entityAggregations.js

```diff
@@ -83,7 +78,8 @@ const sumValueMapper = value => value;
 
 const countValueMapper = () => 1;
 
-const perOrgGroupKey = analytic => `${analytic.dataElement}__${analytic.organisationUnit}`;
+const perOrgGroupKey = (analytic, organisationUnit) =>
+  `${analytic.dataElement}__${organisationUnit}`;
 
-const perPeriodPerOrgGroupKey = analytic =>
-  `${analytic.period}__${analytic.dataElement}__${analytic.organisationUnit}`;
+const perPeriodPerOrgGroupKey = (analytic, organisationUnit) =>
+  `${analytic.period}__${analytic.dataElement}__${organisationUnit}`;
```

Comment:
Just reverting this entityAggregation file back to before cause we don't need `periodType` conversion here anymore